ci-operator/config/stolostron/observability_core_automation/OWNERS

```diff
@@ -12,4 +12,4 @@ approvers:
 options: {}
 reviewers:
 - bjoydeep
-- haoqing0110
+- haoqing0110
```

Comment:
I believe we need to just add a new line at the end of the file to match the one that already exists to resolve this discrepancy.